meson.build

```diff
@@ -4,13 +4,14 @@ project(
     version: '0.4.0', 
     license: 'GNU General Public License v2 (GPLv2)', 
     # license_files: 'LICENSE.txt',
-    meson_version: '>=1.0.0'
+    meson_version: '>=1.0.0',
+    default_options : ['c_std=c11', 'cpp_std=c++11']
 )
 
 poppler_dep = dependency('poppler-cpp', version: '>=0.26.0')
 
 python_mod = import('python')
-python3 = python_mod.find_installation('python3')
+python3 = python_mod.find_installation('python3', pure: false)
```

Comment:
This was already added in #75

Reply:
Yeah, It has to be merged after https://github.com/cbrunet/python-poppler/pull/75. Without this this MR build was failing.